dashboard/test/ui/features/assignment_on_overview_pages.feature

```diff
@@ -17,13 +15,12 @@ Feature: (Un)Assign on script and course overview pages
     And check that the URL contains "courses/csp-2017"
     And I wait until element ".uitest-CourseScript" is visible
     And I wait until element ".uitest-unassign-button" is visible
-    And I wait until element ".uitest-assigned" is visible
     # One assign button for each of 8 currently unassigned units in this course.
     Then the overview page contains 8 assign buttons
     And I press the first ".uitest-unassign-button" element
-    And I wait until element ".uitest-assigned" is not visible
     # One assign button for each of 9 currently unassigned units in this course,
-    # and one at the top of the page indicating the course is assigned.
+    # and one at the top of the page indicating the course is assigned
+    And I wait for the page to fully load
```

Comment:
is the previous step doing a page navigation? if so, the previous step needs to use `to load a new page`, otherwise, this step might check that the page is fully loaded before you've even navigated